### PR TITLE
gnome: update to 45.0

### PR DESCRIPTION
**Pull request by chrysos349**
**MERGE FIRST:**
- #48752
- #48093 
- #49096 
- #48764
- #48985 
- #48839
- #48884 (required by `gnome-tour` and `snapshot`)

**NOTES**
- `loupe` replaces `eog`
- `snapshot` replaces `cheese`
- `epiphany` requires `webkit2gtk>=2.41.1`

#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64)
- I built this PR locally for these architectures (if supported. mark crossbuilds):
  - aarch64-musl x
  - armv7l x
  - armv6l-musl x

[ci skip]

### CODE REVIEW

**Comment by oreo639** [comment]
Many of the applications and libraries can be updated separately. Feel free to submit update PRs for them, assuming they don't rely on e.g. gnome-desktop 45 or smth.

**Comment by oreo639** [comment]
You can merge this PR with gnome-core.

File: srcpkgs/gnome/template [hunk 1]
```diff
@@ -1,99 +1,102 @@
...
+version=45.0
 revision=1
 build_style=meta
 short_desc="GNOME meta-package for Void Linux"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="GPL-2.0-only, LGPL-2.0-only"
 homepage="https://www.gnome.org"
 
+# 'snapshot' replaces cheese
+# 'loupe' replaces gnome-photos
```
Comment (oreo639) [inline srcpkgs/gnome/template:12]: loupe replaces eog, not gnome-photos.

File: srcpkgs/gnome/template [hunk 2]
```diff
@@ -1,99 +1,104 @@
...
 revision=1
 build_style=meta
 short_desc="GNOME meta-package for Void Linux"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="GPL-2.0-only, LGPL-2.0-only"
 homepage="https://www.gnome.org"
 
+# loupe replaces eog since gnome-45.0
+# snapshot replaces cheese since gnome-45.0
+
```
Comment (oreo639) [inline srcpkgs/gnome/template:13]: Also, remove these comments

**Comment by chrysos349** [comment]
> You can merge this PR with gnome-core.

merged with #48752

**Pull request closed without merging**